<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Facility Control</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>시설 제어</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_belt">
     <property name="title">
      <string>벨트 제어</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_belt_speed">
          <property name="text">
           <string>속도 조절:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_belt">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_belt"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_belt_start">
          <property name="text">
           <string>시작</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_belt_stop">
          <property name="text">
           <string>정지</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_belt_emergency">
          <property name="text">
           <string>비상정지</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_gate">
     <property name="title">
      <string>게이트 제어</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>게이트 A:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_gate_a_open">
          <property name="text">
           <string>열기</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_gate_a_close">
          <property name="text">
           <string>닫기</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>게이트 B:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_gate_b_open">
          <property name="text">
           <string>열기</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_gate_b_close">
          <property name="text">
           <string>닫기</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_system">
     <property name="title">
      <string>시스템 상태</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTextEdit" name="textEdit_system_status"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 